apps/web/src/pages/ApplicantDashboardPage/components/CareerDevelopmentTaskCard.tsx

```diff
@@ -0,0 +1,276 @@
+}: CareerDevelopmentTaskCardProps) => {
+  const intl = useIntl();
+  const paths = useRoutes();
+  const functionalCommunityInterestsFragments = getFragment(
+    PreviewListItemFunctionalCommunity_Fragment,
+    communityInterestsQuery,
+  );
+
+  const careerPlanningMetaData: AccordionMetaData[] = [
+    {
+      key: "edit-career-planning-key",
+      type: "link",
+      href: paths.employeeProfile(),
+      color: "secondary",
+      children: (
+        <>
+          {intl.formatMessage({
+            defaultMessage: "Edit career planning",
+            id: "jno96W",
+            description:
+              "Link to a page to edit your career planning information",
+          })}
+        </>
+      ),
+    },
+  ];
+
+  const functionalCommunitiesMetaData: AccordionMetaData[] = [
+    {
+      key: "add-community-key",
+      type: "link",
+      href: "#",
+      color: "secondary",
+      children: (
+        <>
+          {intl.formatMessage({
+            defaultMessage: "Add a community",
+            id: "kBEib8",
+            description: "Link to a page to add a functional community",
+          })}
+        </>
+      ),
+    },
+    {
+      key: "career-planning-key",
+      type: "link",
+      href: paths.employeeProfile(),
+      color: "secondary",
+      children: (
+        <>
+          {intl.formatMessage({
+            defaultMessage: "Career planning",
+            id: "zN7MBv",
+            description: "Title for a users career plan",
+          })}
+        </>
+      ),
+    },
+  ];
+
+  return (
+    <>
+      <div
+        data-h2-display="base(flex)"
+        data-h2-flex-direction="base(column)"
+        data-h2-gap="base(x1)"
+        data-h2-flex-grow="p-tablet(2)"
+      >
+        <TaskCard.Root
+          icon={Cog8ToothIcon}
+          title={intl.formatMessage({
+            defaultMessage: "Career development",
+            id: "+2/ksA",
+            description: "Card title for career development",
+          })}
+          headingColor="secondary"
+          headingAs="h2"
+        >
+          <TaskCard.Item>
+            <Accordion.Root
+              type="multiple"
+              data-h2-padding-bottom="base:selectors[>.Accordion__Item > .Accordion__Content](x.5)"
+            >
+              <Accordion.Item value="your_career_planning">
+                <Accordion.Trigger
+                  as="h3"
+                  subtitle={intl.formatMessage({
+                    defaultMessage:
+                      "Opt in to your profile information being shared with recruiters and talent management staff so that you can be referred for job opportunities or training.",
+                    id: "L4UWvU",
+                    description:
+                      "Subtitle explaining career planning expandable withing career development card",
+                  })}
+                >
+                  {intl.formatMessage({
+                    defaultMessage: "Career planning",
```

Comment:
`zN7MBv` is identical.

Reply:
Yeah, I copy-pasted it from there

Reply:
Yes, but because the description here is different, it has a different id (`E0I0Yj`) and therefore will have two sources of truth for the same string. If the meaning of the string is the same, there should be a common string that is shared to avoid duplication and possible inconsistent translation.